packages/svg_icon_widget/pubspec.yaml

```
@@ -0,0 +1,30 @@
+    sdk: flutter
+
+topics:
+  - svg
+  - icon
+  - svg-icon
+  - flutter-svg
+  - flutter-icon
+
+screenshots:
+  - description: 'SVG Icon Widget Showcase'
+    path: assets/showcase.png
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)